modules/infrastructure/cloud_build_permission/main.tf

```diff
@@ -1,5 +1,9 @@
 # Required to execute cloud build runs with this same service account
 resource "google_project_iam_member" "service_account_user_itself" {
+  # Why: Image scanning is run from inside a container. As it needs to get the image from the registry it needs a token to get it from the registry.
+  # How to avoid security issues: As in the next implementation scanning will be run from within cloudrun which has needed permissions and won't need a token.
+  # Warning: Organization users musn't be able to impersonate as the created service account.
+  #ts:skip=AC_GCP_0006
```

Comment:
non-blocking, but just to enhance understandably, would change order of the comments

ex.:
```
# AC_GCP_006
# Warning: Organization users musn't be able to impersonate as the created service account.
# Why: Image scanning is run from inside a container. As it needs to get the image from the registry it needs a token to get it from the registry.
# How to avoid security issues: As in the next implementation scanning will be run from within cloudrun which has needed permissions and won't need a token.
#ts:skip=AC_GCP_0006
```